C/primitive/elements/primitive.h

```diff
@@ -196,6 +197,7 @@ typedef struct sigInput {
 typedef struct transaction {
   const sigInput* input;
   const sigOutput* output;
+  const sigOutput* const * feeOutputs;
```

Comment:
The `transaction` type is opaque so FFI is not affected, right? We build `transaction` from `rawTransaction` and then pass the opaque struct to the Simplicity runtime, without caring about its contents.

Reply:
Yes. That is the intended API.

Reply:
The intention is that the transaction value can be reused between validating multiple Simplicity inputs within one transaction.